src/behaviors/loginPrompt.js

```diff
@@ -1,35 +1,42 @@
 export default {
-  /**
-   * Init login prompt observer
-   * Called by Polymer on attach
-   * @return {undefined}
-   */
-  attached() {
-    this._promptLoginOnEditable();
+  properties: {
+
+    loginPrompt: {
+      type: Boolean,
+      observer: '_promptLoginOnEditable'
+    }
+
   },
 
   /**
    * Observer to prompt for login if editable && !authed
    * @return {undefined}
    */
-  _promptLoginOnEditable() {
-    let simplaLogin = this.$['login'],
+  _promptLoginOnEditable(loginPrompt) {
+    let { _simplaObservers: observers } = this,
+        simplaLogin = this.$['login'],
         promptLogin = (editable) => {
           if (editable && !this._authenticated) {
-            simplaLogin.prompt()
-              .then(loggedIn => Simpla.editable(loggedIn));
+            simplaLogin.prompt().then(loggedIn => {
+              Simpla.editable(loggedIn)
+            });
           }
         },
         initialPrompt = () => {
+          let shouldPrompt = Simpla.getState('editable') && loginPrompt;
+
           if (typeof simplaLogin.prompt === 'function') {
-            promptLogin(Simpla.getState('editable'));
+            promptLogin(shouldPrompt);
```

Comment:
Re setTimeout - sorry, misread that. cool 👍 

If `promptLogin`'s implementation changes, then the `initialPrompt` checks become useless, so they're pretty tightly bound. Also the only reason there's a check for if it can be done is because `promptLogin`'s implementation may fail at some point, but that's more a concern for `promptLogin` i.e. it should either tell you it can't do it's job, or it should just always be able to do it's job i.e. contain that repeating logic in itself.